### PR TITLE
Aged 16 to 64 years level 3 or above qualifications

### PR DESCRIPTION
**Pull request by bailed22**

### CODE REVIEW

File: src/store/metadata/stub/content/editions/age-16-to-64-years-level_2023-05.json [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+        "table_schema": {
+          "columns": [
+            {
+              "name": "period",
+              "datatype": "datetime",
+              "titles": "Gregorian Time Interval",
+              "description": "The time period associated with the observation."
+            },
+            {
+              "name": "areacd",
```
Comment (osamede20) [inline src/store/metadata/stub/content/editions/age-16-to-64-years-level_2023-05.json:60]: Please check this column mapping (title, etc) with the `aged-16-to-64-years-level-3-or-above-qualifications.json `on https://github.com/GSS-Cogs/family-ess

Also check the column mapping for other dimensions, as well.

Some columns were set to `false`. In that case, we don't need to map those columns

File: src/store/metadata/stub/content/editions/versions/age-16-to-64-years-level_2023-05.json [hunk 2]
```diff
@@ -0,0 +1,88 @@
+{
+    "@context": "https://staging.idpd.uk/#ns",
+    "@id": "https://staging.idpd.uk/datasets/age-16-to-64-years-level/editions/2023-05/versions",
```
Comment (osamede20) [inline src/store/metadata/stub/content/editions/versions/age-16-to-64-years-level_2023-05.json:3]: Please effect all the changes requested on the editions file above

**Review by osamede20 — CHANGES_REQUESTED** [review]

File: src/store/metadata/stub/content/datasets.json [hunk 3]
```diff
@@ -477,8 +477,54 @@
...
+      "spatial_coverage": "E92000001",
+      "temporal_resolution": ["P1Y"],
+      "temporal_coverage": {
+        "start": "2004-01-01T12:00:00Z",
+        "end": "2021-12-31T12:00:00Z"
+      },
+      "editions_url": "https://staging.idpd.uk/datasets/age-16-to-64-years-level/editions",
+      "editions": [
+        {
+          "@id": "https://staging.idpd.uk/datasets/age-16-to-64-years-level/editions/2023-05",
```
[the remark this answers is not in the document]
Comment (bailed22) [reply src/store/metadata/stub/content/datasets.json:521]: done

File: src/store/metadata/stub/content/editions/age-16-to-64-years-level_2023-05.json [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+        "issued": "2023-05-23T10:17:47Z",
+        "modified": "2023-08-27T13:10:00Z",
+        "spatial_resolution":["E06", "E07", "E08", "E09", "E11"],
+        "spatial_coverage": "E92000001",
+        "temporal_resolution": ["P1Y"],
+        "temporal_coverage": {
+          "start": "2004-01-01T12:00:00Z",
+          "end": "2021-12-31T12:00:00Z"
+        },
+        "versions_url": "https://staging.idpd.uk/datasets/age-16-to-64-years-level/editions/2023-05/versions",
```
[the remark this answers is not in the document]
Comment (bailed22) [reply src/store/metadata/stub/content/editions/age-16-to-64-years-level_2023-05.json:43]: done

File: src/store/metadata/stub/content/editions/age-16-to-64-years-level_2023-05.json [hunk 5]
```diff
@@ -0,0 +1,119 @@
+{
+    "@context": "https://staging.idpd.uk/#ns",
+    "@id": "https://staging.idpd.uk/datasets/age-16-to-64-years-level/editions",
```
[the remark this answers is not in the document]
Comment (bailed22) [reply src/store/metadata/stub/content/editions/age-16-to-64-years-level_2023-05.json:3]: good

File: src/store/metadata/stub/content/editions/versions/age-16-to-64-years-level_2023-05.json (hunk 2 above)
Comment (bailed22) [reply src/store/metadata/stub/content/editions/versions/age-16-to-64-years-level_2023-05.json:3]: done

File: src/store/metadata/stub/content/editions/age-16-to-64-years-level_2023-05.json [hunk 6]
```diff
@@ -62,18 +62,6 @@
               "titles": "ONS Geography Code",
               "description": "The statistical geography of the observation being measured."
             },
-            {
-              "name": "areanm",
-              "datatype": "string",
-              "titles": "Area Name",
```
Comment (osamede20) [inline src/store/metadata/stub/content/editions/age-16-to-64-years-level_2023-05.json]: Please revert change and include Area Name
change title to "Statistical Geography"

File: src/store/metadata/stub/content/editions/versions/age-16-to-64-years-level_2023-05.json [hunk 7]
```diff
@@ -31,18 +31,6 @@
                   "titles": "ONS Geography Code",
                   "description": "The statistical geography of the observation being measured."
                 },
-                {
-                  "name": "areanm",
-                  "datatype": "string",
-                  "titles": "Area Name",
```
Comment (osamede20) [inline src/store/metadata/stub/content/editions/versions/age-16-to-64-years-level_2023-05.json]: Please revert change and include Area Name
change title to "Statistical Geography"

File: src/store/metadata/stub/content/editions/age-16-to-64-years-level_2023-05.json [hunk 8]
```diff
@@ -53,7 +53,7 @@
             {
               "name": "period",
               "datatype": "datetime",
-              "titles": "Gregorian Time Interval",
+              "titles": "Period",
```
Comment (osamede20) [inline src/store/metadata/stub/content/editions/age-16-to-64-years-level_2023-05.json:56]: change to "Year"

File: src/store/metadata/stub/content/editions/versions/age-16-to-64-years-level_2023-05.json [hunk 9]
```diff
@@ -9,20 +9,20 @@
...
-            "download_url": "https://staging.idpd.uk/datasets/age-16-to-64-years-level/editions/2023-05/versions/1.csv",
+            "download_url": "https://staging.idpd.uk/datasets/age-16-to-64-years-level-3-or-above-qualifications/editions/2023-05/versions/1.csv",
             "media_type": "text/csv",
             "table_schema": {
               "columns": [
                 {
                   "name": "period",
                   "datatype": "datetime",
-                  "titles": "Gregorian Time Interval",
+                  "titles": "Period",
```
Comment (osamede20) [inline src/store/metadata/stub/content/editions/versions/age-16-to-64-years-level_2023-05.json:25]: change to "Year"

**Review by osamede20 — DISMISSED** [review]

File: src/store/metadata/stub/content/editions/versions/age-16-to-64-years-level_2023-05.json (hunk 9 above)
Comment (bailed22) [reply src/store/metadata/stub/content/editions/versions/age-16-to-64-years-level_2023-05.json:25]: noted

File: src/store/metadata/stub/content/editions/age-16-to-64-years-level_2023-05.json (hunk 6 above)
Comment (bailed22) [reply src/store/metadata/stub/content/editions/age-16-to-64-years-level_2023-05.json]: request commited

**Review by GregoryJPavier — COMMENTED** [review]
An issue with the CSV, but the metadata looks good.

**Review by GregoryJPavier — APPROVED** [review]

**Comment by bailed22** [comment]
The file already been deleted off the datasets because its not
reflecting on the main after been merge.

**Pull request merged**